### PR TITLE
Fixed UI regression

### LINKED ISSUE #15571: Regression: Disabling of Extension installation regressed for specific build
- Azure Data Studio Version: 1.29

#15320 - this change may have caused the regression

Under SAW build Menu Item "Install Extension from VSIX Package" along with command line option to do this need to be disabled. This regressed after the above change.

### PR DESCRIPTION
**Pull request by rajeshka**
This PR fixes #15571

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: "Code Scanning - Action"
```
Comment (alanrenmsft) [inline .github/workflows/codeql.yml:1]: why is this file included in the pr?

File: src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts [hunk 2]
```diff
@@ -74,6 +74,7 @@ import { EditorExtensions } from 'vs/workbench/common/editor';
 import { WORKSPACE_TRUST_EXTENSION_SUPPORT } from 'vs/workbench/services/workspaces/common/workspaceTrust';
 import { IStorageService, StorageScope, StorageTarget } from 'vs/platform/storage/common/storage'; // {{SQL CARBON EDIT}}
 import * as locConstants from 'sql/base/common/locConstants'; // {{SQL CARBON EDIT}}
+import product from 'vs/platform/product/common/product';
```
Comment (alanrenmsft) [inline src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts:77]: missing the {{ SQL CARBON EDIT}} comment

Comment (alanrenmsft) [reply src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts:77]: same for the changes below.

File: .github/workflows/codeql.yml (hunk 1 above)
Comment (rajeshka) [reply .github/workflows/codeql.yml:1]: This should not been included. Let me fix it

**Review by alanrenmsft — APPROVED** [review]
